### PR TITLE
Add GitHub Actions

### LINKED ISSUE #145: Replace TravisCI with GitHub Actions

### PR DESCRIPTION
**Pull request by owenjonesuob**
Replace Travis/appveyor with GitHub Actions (closes #145).

Plus a slight tidy-up of the NEWS file, .Rbuildignore etc

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -15,11 +15,12 @@ knitr::opts_chunk$set(
 
 # goodpractice <img src="man/figures/logo.png" align="right" width="20%" height="20%" />
 
-[![Linux Build Status](https://travis-ci.org/MangoTheCat/goodpractice.svg?branch=master)](https://travis-ci.org/MangoTheCat/goodpractice)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/github/MangoTheCat/goodpractice?svg=true)](https://ci.appveyor.com/project/gaborcsardi/goodpractice)
-[![](https://www.r-pkg.org/badges/version/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
-[![CRAN RStudio mirror downloads](https://cranlogs.r-pkg.org/badges/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
```
Comment (ddbortoli) [inline README.Rmd:21]: Any particular reason to remove this badge?

**Review by ddbortoli — COMMENTED** [review]
All looking great, thanks for that! I just left a very small comment on removing one of the badges, if we're splitting hairs 😅

File: README.Rmd (hunk 1 above)
Comment (owenjonesuob) [reply README.Rmd:21]: Oh!! No, that's removed in error - it did catch my eye yesterday but I promptly forgot about it 😁 I'll reinstate it now!

**Review by ddbortoli — APPROVED** [review]
Nice! :)

**Pull request merged**